Orange/widgets/visualize/owscatterplot.py

```diff
@@ -428,19 +392,13 @@ def set_attr(self, attr_x, attr_y):
             self.attr_changed()
 
     def attr_changed(self):
-        self.cb_reg_line.setEnabled(self.can_draw_regresssion_line())
         self.setup_plot()
         self.commit()
 
     def setup_plot(self):
         super().setup_plot()
-        for axis, var in (("bottom", self.attr_x), ("left", self.attr_y)):
-            self.graph.set_axis_title(axis, var)
-            if var and var.is_discrete:
-                self.graph.set_axis_labels(axis,
```

Comment:
set_axis_labels() should probably be removed from OWScatterPlotGraph

Reply:
Removed.